src/mumble/UserModel.cpp

```diff
@@ -1271,6 +1271,10 @@ void UserModel::setComment(Channel *c, const QString &comment) {
 					QToolTip::showText(QCursor::pos(), data(index(c, 0), Qt::ToolTipRole).toString(),
 									   Global::get().mw->qtvUsers);
 				}
+			} else if (c->iId == static_cast< unsigned int >(~iChannelDescription)) {
+				iChannelDescription               = -1;
+				Global::get().mw->cContextChannel = c;
+				QTimer::singleShot(0, Global::get().mw, SLOT(on_qaChannelDescriptionView_triggered()));
```

Comment:
Using a function pointer should be preferred over the `SLOT` macro (I think there should be a suitable overload for that).

Reply:
Okay, should I change the "View Comment" context action as well? It's implemented the same way.

Reply:
If you feel like it, the change would be welcome (as a separate `REFAC` commit) but it's not necessary. Our strategy is to fix this kind of stuff for new code or when old code is modified anyway (as otherwise we'd have to spend a lot of time and effort for all the refactorings)